README.md

```diff
@@ -131,8 +131,6 @@ You can generate a data migration as you would a schema migration:
     rake db:migrate:up:with_data      # Runs the "up" for a given migration VERSION
     rake db:migrate:with_data         # Migrate the database data and schema (options: VERSION=x, VERBOSE=false)
     rake db:rollback:with_data        # Rolls the schema back to the previous version (specify steps w/ STEP=n)
-    rake db:schema:load:with_data     # Load both schema.rb and data_schema.rb files into the database
```

Comment:
Since these do not do anything with the data migrations, I think they shouldn't be in the readme. Thoughts? @ilyakatz